checker.go

```diff
@@ -15,6 +15,19 @@ func cleanMsg(m map[string]interface{}) map[string]interface{} {
 	if _, ok := m["context"].(map[string]interface{}); ok {
 		m["context"] = delete(m["context"].(map[string]interface{}), "library")
 	}
+	// delete empty fields
+	for k, v := range m {
+		field, ok := v.(map[string]interface{})
```

Comment:
@f2prateek we can actually check using type assertion - this should be stricter now.